src/javaExercises/primeNumbers/PrimeNumbers.java

```diff
@@ -0,0 +1,34 @@
+package javaExercises.primeNumbers;
+
+import java.util.Scanner;
+
+public class PrimeNumbers {
+    public static boolean isPrime(int n) {
+        if (n == 1)
+            return false;
+        for (int i = 2; i < n; i++) {
+            if (n % i == 0)
+                return false;
+        }
+        return true;
+    }
+
+    public static void printPrimeNumbers(int n) {
+        for (int i = 2; i < n; i++) {
+            if (isPrime(i))
+                System.out.print(+i + ", ");
```

Comment:
what is the need of **+** sign places before **i**?

Reply:
Done. My mistake